src/jdk.crypto.ec/share/classes/sun/security/ec/XDHPublicKeyImpl.java

```diff
@@ -76,7 +76,7 @@ public final class XDHPublicKeyImpl extends X509Key implements XECPublicKey {
         // clear the extra bits
         int bitsMod8 = params.getBits() % 8;
         if (bitsMod8 != 0) {
-            int mask = (1 << bitsMod8) - 1;
+            byte mask = (byte) ((1 << bitsMod8) - 1);
```

Comment:
I may not insert a whitespace after the "(byte)" conversion.  Similar to following update.  Otherwise, looks good to me.